docs/build/get-started.md

```diff
@@ -260,18 +260,18 @@ Change into your new *hello-world* directory and have a look around:
 tree -L 2 -a
 
 .
-├── .fluence              # this is where Fluence CLI internals are kept including schemas and project secrets
-│   ├── aqua
-│   ├── schemas
-│   └── workers.yaml
+├── .fluence              # Fluence CLI internals, including schemas and project secrets
+│   ├── aqua              # Generated Aqua code, including Service APIs and deployment info
```

Comment:
still doing that?

Reply:
also, line 35 should be updated to binary install.

Reply:
still doing what?

Reply:
aqua code in .fluence

Reply:
> also, this looks as good a time as any to update line 35 to binary install.

@folex what do you think about this? Let's update the installation method?